.travis.yml

```diff
@@ -6,6 +6,7 @@ rvm:
 before_script:
   - psql -c 'create database travis_ci_test;' -U postgres
   - cp config/database.yml.travis spec/dummy/config/database.yml
+  - cp spec/dummy/config/air.yml.example spec/dummy/config/air.yml
```

Comment:
I remove this file due to https://github.com/dice-cyfronet/atmosphere/pull/15#discussion-diff-22999352